pkgs/tools/typesetting/soupault/default.nix

```diff
@@ -16,7 +16,7 @@ ocamlPackages.buildDunePackage rec {
     owner = "PataphysicalSociety";
     repo = pname;
     rev = version;
-    sha256 = "sha256-txNKAZMd3LReFoAtf6iaoDF+Ku3IUNDzBWUqGC2ePKw=";
+    sha256 = "sha256-aFvnq+taLUFKYnO4SQT6TGH5lEDAHlMqNmtdLwFpkj4=";
```

Comment:
Is this hash correct?

Reply:
It’s also my opinion. I do not doubt that you’ve witnessed a state of the Universe in which this hash was correct. But I humbly suggest that something unexpected may have happened, leading to the following kind of error:

> unpacking source archive /private/tmp/nix-build-source.drv-0/4.0.1.tar.gz
error: hash mismatch in fixed-output derivation '/nix/store/68f7yq4kj59sd8mp76rjngh6jkxmxqlm-source.drv':
         specified: sha256-aFvnq+taLUFKYnO4SQT6TGH5lEDAHlMqNmtdLwFpkj4=
            got:    sha256-Ojsup/BgXsqhRUsU6JevvuLnMtVs2Kd5kTeDvVO5BOk=

This is why I ask; and maybe, it would prove useful if you take the time to double-check that the hash in your commit is indeed what you want to add into `nixpkgs`.

Thanks for your contribution.

Reply:
Yes, I have this now

```console
$ git diff                                                              
diff --git a/pkgs/tools/typesetting/soupault/default.nix b/pkgs/tools/typesetting/soupault/default.nix
index 9954bf6f67b..ec5bcd6c3b3 100644
--- a/pkgs/tools/typesetting/soupault/default.nix
+++ b/pkgs/tools/typesetting/soupault/default.nix
@@ -14,7 +14,7 @@ ocamlPackages.buildDunePackage rec {
     owner = "PataphysicalSociety";
     repo = pname;
     rev = version;
-    sha256 = "sha256-aFvnq+taLUFKYnO4SQT6TGH5lEDAHlMqNmtdLwFpkj4=";
+    sha256 = "sha256-Ojsup/BgXsqhRUsU6JevvuLnMtVs2Kd5kTeDvVO5BOk=";
   };
```

---

I would have prefer that comment phrased as: "This isn't the hash I got. Can you double check that nothing has changed since you last built it?". "Is this hash correct?" seems almost rhetorical without signalling to me why you saw something as amiss.

Reply:
It seems the owner pushed over the tag to get the CHANGELOG included after I asked if the project could get one so I could fill out that part of the metadata.